web3-dev/smart-contracts/sc-execution.rst

```diff
@@ -0,0 +1,68 @@
+========================
+Smart contract execution
+========================
+
+Introduction
+============
+
+In Massa you can execute smart contracts not only deploy and call.
```

Comment:
I think this sentence is confusing.

Reply:
I have written it again. Tell me if it's better